shared/js/devtools/panel.es6.js

```diff
@@ -0,0 +1,217 @@
+const table = document.querySelector('#request-table')
+const clearButton = document.getElementById('clear')
+const refreshButton = document.getElementById('refresh')
+const protectionButton = document.getElementById('protection')
+const tabPicker = document.getElementById('tab-picker')
+const tdsOption = document.getElementById('tds')
+const displayFilters = document.querySelectorAll('#table-filter input')
+
+let tabId = chrome.devtools?.inspectedWindow?.tabId || parseInt(0 + new URL(document.location.href).searchParams.get('tabId'))
+const port = chrome.runtime.connect()
+const features = [
+    'canvas',
```

Comment:
These will need to be renamed since brad's project has gone live. Perhaps there should be a mapping of short name to the long names?

Like this: https://github.com/duckduckgo/privacy-configuration/pull/1/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R53-R65